<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SerialPortlxyClass</class>
 <widget class="QMainWindow" name="SerialPortlxyClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>512</width>
    <height>383</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MySerialPort</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_5">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>接收窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QTextBrowser" name="textBrowser_receive">
            <property name="html">
             <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'SimSun'; font-size:9pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Defalut values:&lt;/p&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;123456789&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>串口号</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_SPnum"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>波特率</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_Baudrate"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>数据位</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_databit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>奇偶校验</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_exambit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>停止位</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QComboBox" name="comboBox_stopbit"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QPushButton" name="pushButton_scan">
            <property name="text">
             <string>扫描串口</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_open">
            <property name="text">
             <string>打开串口</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_save">
            <property name="text">
             <string>保存数据</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_clear">
            <property name="text">
             <string>清空数据</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_10">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>发送窗口</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QTextEdit" name="textEdit_send"/>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4">
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_load">
              <property name="text">
               <string>打开文件</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_4">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton__clearsend">
              <property name="text">
               <string>清空发送</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="verticalSpacer_3">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_send">
              <property name="text">
               <string>发送数据</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="SerialPortlxy.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>pushButton_open</sender>
   <signal>clicked()</signal>
   <receiver>SerialPortlxyClass</receiver>
   <slot>slot_open()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>738</x>
     <y>240</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_save</sender>
   <signal>clicked()</signal>
   <receiver>SerialPortlxyClass</receiver>
   <slot>slot_save()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_clear</sender>
   <signal>clicked()</signal>
   <receiver>SerialPortlxyClass</receiver>
   <slot>slot_clear()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>788</x>
     <y>299</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_load</sender>
   <signal>clicked()</signal>
   <receiver>SerialPortlxyClass</receiver>
   <slot>slot_load()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>402</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton__clearsend</sender>
   <signal>clicked()</signal>
   <receiver>SerialPortlxyClass</receiver>
   <slot>slot_clearsend()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_send</sender>
   <signal>clicked()</signal>
   <receiver>SerialPortlxyClass</receiver>
   <slot>slot_send()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>787</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton_scan</sender>
   <signal>clicked()</signal>
   <receiver>SerialPortlxyClass</receiver>
   <slot>scan_SP()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>688</x>
     <y>95</y>
    </hint>
    <hint type="destinationlabel">
     <x>424</x>
     <y>302</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>slot_clear()</slot>
  <slot>slot_send()</slot>
  <slot>slot_load()</slot>
  <slot>slot_save()</slot>
  <slot>slot_open()</slot>
  <slot>slot_clearsend()</slot>
  <slot>scan_SP()</slot>
 </slots>
</ui>
